python/pyspark/pandas/utils.py

```diff
@@ -467,11 +467,18 @@ def is_testing() -> bool:
 
 def default_session() -> SparkSession:
     spark = SparkSession.getActiveSession()
-    if spark is not None:
-        return spark
+    if spark is None:
+        spark = SparkSession.builder.appName("pandas-on-Spark").getOrCreate()
+
+    if spark.conf.get("spark.sql.ansi.enabled"):
```

Comment:
Currently we always see the warning for the first running this.
This should be `spark.conf.get("spark.sql.ansi.enabled", "false") == "true"`.
cc @itholic, @HyukjinKwon

Reply:
Let's make sure testing this especially when there's no test next time ...